server/src/stateManagement.mjs

```diff
@@ -130,6 +130,23 @@ function handleSequenceOfResponseValues(userId, methodName, params, resp, userSt
   return resp;
 }
 
+// Log Error for invalid methodName
+function logError(methodName, resultErrors, resp) {
```

Comment:
Can we change this name to be more specific to what it's logging?

Reply:
Taken care